save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/AvatarForm.kt

```diff
@@ -0,0 +1,64 @@
+/**
+ * Function component for avatar form
+ */
+
+@file:Suppress("FILE_NAME_MATCH_CLASS")
+
+package com.saveourtool.save.frontend.components.basic
+
+import com.saveourtool.save.frontend.components.modal.modalAvatarBuilder
+import com.saveourtool.save.frontend.externals.imageeditor.reactAvatarImageCropper
+import csstype.ClassName
+import csstype.rem
+import js.core.jso
+import react.FC
+import react.Props
+import react.dom.html.ReactHTML
+import web.file.File
+
+val avatarForm = avatarForm()
+
+/**
+ * AvatarForm component props
+ */
+external interface AvatarFormProps : Props {
+    /**
+     * Flag to handle avatar Window
+     */
+    var isOpen: Boolean
+
+    /**
+     * Callback to update state for close window.
+     */
+    var onCloseWindow: (Boolean) -> Unit
```

Comment:
do we really need a boolean as argument here?

Reply:
Removed